data/hai/hai reveal 1 intro.txt

```diff
@@ -11,47 +11,535 @@
+			
+			choice
+				`	"I'll leave you in peace."`
+					goto peace
+				`	"Will he be ok?"`
+					goto "him ok"
+				`	"Are you ok?"`
+					goto "she ok"
+				`	"You're his aunt?"`
+			`	She gives you a strange look, as if unsure whether to dignify that with a response, and then answers. "His mother was in the militia during the war. She died. His father did not survive that loss. He came to be looked after by his uncle who had heard rumors of the Hai, and couldn't afford to both look after him and pay his debts in the Free Worlds. So they came here and found me."`
+				goto uncle
+			label "him ok"
+			`	"Rightfully, I do not know."`
+				goto uncle
+			label "she ok"
+			`	"I will be fine, though it is challenging adapting to how humans deal with crisis and trauma," she says. "No doubt they will be having a far worse time of things on Allhome. Still..."`
+			
+			label uncle
+			`	She looks wistfully towards the docks. "His uncle is meant to be returning from having gone to pay off the final sum on his debt and wrap up some final business in human space. With the current events, we are both very worried he will not return, though I am doing my best to pretend otherwise."`
+			`	She settles back in her seat with deliberate dignity.`
+				goto leave
+				
+			label peace
+			`	"Thank you, but it is okay, Captain.`
+			label leave
+			`	"Do not trouble yourself over having questioned us. There is no shortcut for processing trauma, and it does the boy no good to hide from the world."`
+			`	She gestures at the screen. "No doubt many people will have many ideas for how to help things soon enough. If it troubles you, lend them a hand. Now, be about your business and farewell."`
+			label walking
+			`	Barely ten minutes later there is already someone on the screen arguing for a range of measures to help with the pirate situation.`
+				decline
+
+
+
+mission "HR: Call for Aid with Sandsnap"
+	name "Join an effort on <planet>"
+	job
+	repeat
+	description "Experienced combat pilots are requested to apply at the spaceport on <destination> for a project."
+	to offer
+		not "hr: sandsnap project"
+	source
+		government "Hai"
+		not planet "Makerplace"
+	destination "Makerplace"
+	to complete
+		has "hr: sandsnap project"
+	on complete
+		payment 1000 150
+
+
+
+mission "Hai Reveal Concern Period: Reactivating Sandsnap"
+	priority
+	name "Transport the Stationkeeper to Sandsnap."
+	description "Travel to <destination> and safely deliver the Stationkeeper to reactivate the station."
+	source "Makerplace"
+	destination "Sandsnap"
+	passengers 1
+	to offer
+		has "hr: allhome raided"
+	on offer
+		conversation
+			`	Some Hai would like to reopen the Sandsnap station in Heia Due as a better place for coordinating security efforts from after the raid on Allhome. The first mission is to go there with the motley hired guns and the Stationkeeper to reactivate it and make sure it's in working order.`
```

Comment:
```suggestion
			`	Upon entering the spaceport you are immediately greeted by a worried hai, "You must be here for the project," he says, not giving you time to respond. He starts walking at a quick pace, with a gesture encouraging you to follow him. "Times are dire, it has been at least a hundred years since we have had such a threat to our way of life! You must have noticed the increased activity. Perhaps our brethren north were right in assuming evil intents."`
			`	He stops himself, "I must be boring you with politics, let us cut to the matter at hand, shall we?"`
			choice
				`	(Nod.)`
					goto border
				`	"Not at all, I love politics."`
			`	He seems surprised, "Oh. Well in that case, let me explain more of it to you. The Unfettered have been militarizing a lot in the last 20 years, forcing us to do the same in order to protect merchants from their raids. They may be right that we have grown weak, demonstrated by how easy those pirate raids were; we were simply not ready for them. But I will do everything to ensure this will not happen again. Mark my words, captain <last>, the Hai will wake up.`
			label border
			`	"The raids have proven that we are not prepared for an offensive on our borders. In order to address this, we want to repair the abandoned station of <destination>. We have already hired a few mercenaries to help with the defense of our station, most of them posing as human merchants. Once operational, the station should be a major strategic advantage for us, allowing fast resupply and repairs of ships." He stops in front of a very distinguished house, "This is the residence of our esteemed Stationkeeper. You will need to escort him to the station so he may make the necessary adjustments and assess exactly what kind of supplies are required. I'll leave you in his company," he says, heading of at a quick pace.`
			choice
				`	(Greet the Stationkeeper.)`
					goto greet
				`	(Bow before the Stationkeeper.`
			`	She seems flattered, "There is no need, captain, my station is no longer operational, and the same goes for the honor that was due to me. For now, anyway."`
			label greet
			`	You introduce yourself, "You may refer to me simply as the Stationkeeper. Now, if you don't mind, I have been waiting for this opportunity all my life, so I would really like to get going!" You show her to her bunks. She does not seem to be the talkative kind.`
```